overlord/snapstate/snapstate_test.go

```diff
@@ -3723,7 +3723,7 @@ func (s *snapmgrTestSuite) TestEnsureRefreshesAtSeedPolicy(c *C) {
 	c.Check(err, IsNil)
 }
 
-func (s *snapmgrTestSuite) TestEnsureRefreshesAtSeedPolicyNopAtPreesed(c *C) {
+func (s *snapmgrTestSuite) TestEnsureRefreshesAtSeedPolicyNopAtPreseed(c *C) {
```

Comment:
Was this intended to be a `Not` as well?
```suggestion
func (s *snapmgrTestSuite) TestEnsureRefreshesAtSeedPolicyNotAtPreseed(c *C) {
```

Reply:
Ah I thought that might be the case, but usually see it as noop/no-op so asked anyway. Thanks